src/pois.rs

```diff
@@ -45,6 +44,151 @@ static NON_SEARCHABLE_ITEMS: Lazy<BTreeSet<(String, String)>> = Lazy::new(|| {
+        ("BE", vec!["fr", "de", "nl"]),
+        // brazil
+        ("BR", vec!["pt"]),
+        // bulgaria
+        ("BG", vec!["bg"]),
+        // canada
+        ("CA", vec!["en", "fr"]),
+        // china
+        ("CN", vec!["zh"]),
+        // croatia
+        ("HR", vec!["hr"]),
+        // czechia
+        ("CZ", vec!["cs"]),
+        // denmark
+        ("DK", vec!["da"]),
+        // estonia
+        ("EE", vec!["et"]),
+        // france
+        ("FR", vec!["fr"]),
+        // germany
+        ("DE", vec!["de"]),
+        // greece
+        ("GR", vec!["el"]),
+        // ireland
+        ("IE", vec!["ga", "en"]),
+        // italy
+        ("IT", vec!["it"]),
+        // japan
+        ("JP", vec!["ja"]),
+        // south korea
+        ("KR", vec!["ko"]),
+        // latvia
+        ("LV", vec!["lv"]),
+        // lithuania
+        ("LT", vec!["lt"]),
+        // luxembourg
+        ("LU", vec!["lb", "fr", "de"]),
+        // mexico
+        ("MX", vec!["es"]),
+        // moldova
+        ("MD", vec!["ro"]),
+        // netherlands
+        ("NL", vec!["nl"]),
+        // new zealand
+        ("NZ", vec!["en", "mi"]),
+        // north macedonia
+        ("MK", vec!["mk", "sq"]),
+        // norway
+        ("NO", vec!["no"]),
+        // poland
+        ("PL", vec!["pl"]),
+        // portugal
+        ("PT", vec!["pt"]),
+        // romania
+        ("RO", vec!["ro"]),
+        // russia
+        ("RU", vec!["ru"]),
+        // serbia
+        ("RS", vec!["sr"]),
+        // singapour
+        ("SG", vec!["en", "ms", "ta"]),
+        // slovakia
+        ("SK", vec!["sk"]),
+        // slovenia
+        ("SL", vec!["sl"]),
+        // spain
+        ("ES", vec!["es"]),
+        // sweden
+        ("SE", vec!["sv"]),
+        // switzerland
+        ("CH", vec!["de", "fr", "it", "rm"]),
+        // thailand
+        ("TH", vec!["th"]),
+        // tunisia
+        ("TN", vec!["ar"]),
+        // turkey
+        ("TR", vec!["tr"]),
+        // ukraine
+        ("UA", vec!["uk"]),
+        // united kingdom
+        ("GB", vec!["en"]),
+        // usa
+        ("US", vec!["en"]),
+        // uruguay
+        ("UY", vec!["es"]),
+        // uzbekistan
+        ("UZ", vec!["uz"]),
+        // venezuela
+        ("VE", vec!["es"]),
+        // viet nam
+        ("VN", vec!["vi"]),
+    ]
+    .iter()
+    .map(|(a, b)| (a.to_string(), b.clone()))
+    .collect()
+});
+
+fn format_i18n_label<'a>(
+    nice_name: &str,
+    mut admins: impl Iterator<Item = &'a mimir::Admin> + Clone,
+    _country_codes: &[String], // Note: for the moment the country code is not used, but this could change
+    lang: &str,
+) -> String {
+    admins.find(|adm| adm.is_city()).map_or_else(
+        || nice_name.to_string(),
+        |adm| {
+            let local_admin_name = &adm.names.get(lang).unwrap_or(&adm.name);
+            format!("{} ({})", nice_name, local_admin_name)
+        },
+    )
+}
+
+fn format_international_poi_label<'a>(
```

Comment:
If we are using a "country -> languages" map, it's probably redundant to override this function.
The changes you made in `locate_poi` should be enough, no ?

Reply:
I guess so.